op-mode-definitions/show-users.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+    <node name="users">
+      <properties>
+        <help>Show user information</help>
+      </properties>
+      <command>who -H</command>
+      <children>
+        <node name="recent">
```

Comment:
is this command needed? wouldn't it be wiser to add " | tail "   or something like that to only list the latest

Reply:
You mean "show users" or "show users recent"? I'd keep them both I think, and don't underestimate the number of people coming from platforms that lack |tail :)

Reply:
👍  i ment `show users recent`. but i'll let them be there.